docs/UserGuide.md

```diff
@@ -288,7 +286,7 @@ You can add the prefixes in any order you want!
 
 ### Listing all events: `list event`
 
-Shows a list of all events in TalentHub.
+This command allows us to view a list of all events on TalentHub!
```

Comment:
Maybe "us" should be you for word consistency?

Reply:
Got it!